src/components/speaker/SpeakerList.jsx

```diff
@@ -1,25 +1,30 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { Link } from 'react-router-dom'
+import Row from 'react-bootstrap/Row'
+import Col from 'react-bootstrap/Col'
+import Container from 'react-bootstrap/esm/Container'
+import SpeakerItem from './SpeakerItem'
 
-const SpeakerList = ({ speakers }) => (
-  <div className='speakers'>
-    {speakers.map(speaker => (
-      <div key={speaker.id} className='speaker'>
-        <div className='title'>
-          <Link to={`/speaker/${speaker.id}`}>
-            {speaker.title}
-          </Link>
-        </div>
+const SpeakerList = ({ speakers, title }) => (
+  <Container className='my-md-5'>
+    <Row className='speakers'>
+      <Col xs={12} className='my-5'>
+        <h1>{title ? 'Speaker' : title}</h1>
```

Comment:
I think you mean`{title ? title : 'Speakers'}` here to show `title` when it's not empty and 'Speakers' otherwise.